manifests/s/squidowl/halloy/0.0.2024.6/squidowl.halloy.locale.en-US.yaml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: squidowl.halloy
+PackageVersion: 0.0.2024.6
+PackageLocale: en-US
+Publisher: squidowl
+PublisherUrl: https://squidowl.org/
+PublisherSupportUrl: https://halloy.squidowl.org/
+PackageName: Halloy
+PackageUrl: https://github.com/squidowl/halloy/releases/tag/2024.6
+License: GPLv3
```

Comment:
```suggestion
License: GPL-3.0-or-later
```

Reply:
Hi, i'm not an license expert, but the license of Halloy is GPL-3.0 (see here: https://github.com/squidowl/halloy/blob/main/LICENSE). Maybe it could wrong to add the "or-later" extension.

Reply:
Line 637-640 mentions that the license can be gpl-3.0 or later.
I suggested this format as it is used by [ScoopInstaller](https://github.com/ScoopInstaller) manifests which follow spdx short identifier https://spdx.org/licenses/GPL-3.0-or-later.html